README.md

```diff
@@ -76,7 +86,7 @@ The two above might be also used together:
 The action caches the baseline rustdoc for each package in the workspace. The keys used to distinguish the caches consist of four components:
 
  - `prefix-key` input, which defaults to an empty string,
- - `shared-key` input if provided, otherwise a concatenation of the value of environmental variable `GITHUB_JOB`, the value of `package` and the hashed value of the `manifest-path` variable (not the file it points to),
+ - `shared-key` input if provided, otherwise a concatenation of the value of environmental variable `GITHUB_JOB`, the values of `package`, `exclude` and the hashed value of the `manifest-path` variable (not the file it points to),
```

Comment:
```suggestion
 - `shared-key` input if provided, otherwise a concatenation of the value of environmental variable `GITHUB_JOB`, the values of `package` and `exclude` if they are set, and the hashed value of the `manifest-path` variable (not the file it points to),
```
The comma threw me off because it made it seem like `exclude` is related to `manifest-path` more strongly than `package`.